bucketeer/src/main/kotlin/io/bucketeer/sdk/android/internal/evaluation/EvaluationInteractor.kt

```diff
@@ -111,4 +209,10 @@ internal class EvaluationInteractor(
     val evaluations = evaluations[userId] ?: return null
     return evaluations.firstOrNull { it.featureId == featureId }
   }
+
+  companion object {
+    private const val PREFERENCE_KEY_FEATURE_TAG = "bucketeer_feature_tag"
+    private const val PREFERENCE_KEY_EVALUATED_AT = "bucketeer_evaluated_at"
+    private const val PREFERENCE_KEY_USER_ATTRIBUTES_UPDATED = "bucketeer_user_attributes_updated"
```

Comment:
please add these constants to `Constants` as we do for other SharedPreferences keys

Reply:
applied in https://github.com/bucketeer-io/android-client-sdk/pull/88/commits/ce54007dd0f2ff206001e38047b8eefa1b3f53ef